src/utils/UseAxios.test.tsx

```diff
@@ -0,0 +1,131 @@
+import { renderHook } from '@testing-library/react-hooks';
+import { useAxios, ApiResponse } from './UseAxios';
+import { HttpMethods } from './HttpMethods';
+import MockAdapter from 'axios-mock-adapter';
+import axios from 'axios';
+
+const httpHeaders = new Headers({
+  'Content-Type': 'json',
+  'Project-Name': 'sdc',
+});
+
+const getResponseData = { message: 'Testing successful response.' };
+const successfulGetResponse: ApiResponse = {
```

Comment:
It may organize the code a bit more to put these objects into the tests where they are used rather than as global objects? Are these objects used in multiple tests and required to be global? If so, they should be in a respective global describe block.